src/Control/Email/Email.php

```diff
@@ -604,13 +614,16 @@ public function addAttachment($path, $alias = null, $mime = null)
     }
 
     /**
+     * @deprecated 4.12.0 Will be replaced with Symfony\Component\Mime\Email::attach()
+     *
      * @param string $data
      * @param string $name
      * @param string $mime
      * @return $this
      */
     public function addAttachmentFromData($data, $name, $mime = null)
     {
+        Deprecation::notice('4.12.0', 'Will be replaced with Symfony\Component\Mime\Email::attach()');
```

Comment:
```suggestion
        Deprecation::notice('4.12.0', 'Use attach() instead');
```

Reply:
The CMS 5 PR still has this method. Are we removing it or not?

Reply:
Not removing, have removed deprecation